tfjs-layers/src/layers/preprocessing/image_preprocessing.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright 2022 CodeSmith LLC
```

Comment:
Everything else in the TFJS repository uses "Copyright Google Inc." in the license header, but I'm not sure that applies in this case. I've asked our legal team for advice on what to do here and I'll update you when I hear back from them.

Reply:
Thank you for the review! We will make all the necessary changes and resubmit

Reply:
I heard back from our legal team, and they say there's no problem with CodeSmith listing themselves on any new files they write.